source/deploy/postgres-migration.rst

```diff
@@ -229,10 +268,12 @@ Once we set the schema to a desired state, we can start migrating the **data** b
         type tinyint when (<= precision 4) to boolean using tinyint-to-boolean,
         type json to jsonb drop typemod
 
-   EXCLUDING TABLE NAMES MATCHING ~<IR_>, ~<focalboard>, 'schema_migrations', 'db_migrations'
+   EXCLUDING TABLE NAMES MATCHING ~<IR_>, ~<focalboard>, 'schema_migrations', 'db_migrations', 'db_lock',
+        'configurations', 'configurationfiles', 'db_config_migrations'
 
    BEFORE LOAD DO
         $$ ALTER SCHEMA public RENAME TO {{ .source_schema }}; $$,
+        $$ TRUNCATE TABLE {{ .source_schema }}.systems; $$,
```

Comment:
I'd like to confirm why we need to do this. Ideally, the systems table should be empty. And if the customer is re-running the migration, then why not truncate all other tables?

Reply:
No, the systems table is getting populated actually, checked and CRT migrations add values there.

Reply:
It was actually an error by mylsef, we had to remove those from the Systems table before the migration. Because those are added with schema migrations: https://github.com/mattermost/mattermost/blob/master/server/channels/db/migrations/postgres/000054_create_crt_channelmembership_count.up.sql and https://github.com/mattermost/mattermost/blob/master/server/channels/db/migrations/postgres/000055_create_crt_thread_count_and_unreads.up.sql

Reply:
Ahhh okay okay. Nice catch.